### PR TITLE
YouTube embeds with start time work

### PR DESCRIPTION
**Pull request by pageben**
Modified youtube embed code to work with query strings for start time.

### CODE REVIEW

**Comment by pageben** [comment]
Fixed the annoying issue I mentioned in my previous comment with the latest commit.

File: static/index.js [hunk 1]
```diff
@@ -290,6 +304,10 @@ $(document).ready(function(){
         $("#createroom .roomname").val("");
     });
 
+    $("#clearbtn").on('click', function() {
+      $("#messages li").addClass("hidden");
+    });
```
Comment (blamarche) [inline static/index.js:316]: This should only clear the current room. I think it'd still work this way, but better to be 'more correct' and specify the room id in the selector. Resubmit request when changed

File: static/index.js [hunk 2]
```diff
@@ -436,8 +454,16 @@ $("#btnroomusers").on('click', function() {
...
+    var atBottom = false;
+    if($(window).scrollTop() + $(window).height() == $(document).height()) {
+      atBottom = true;
+    }
     appendChatMessage(d.uid,d.room,d.name,d.nick,d.m,d.msgid,d.avatar,d.time);
-    scrollToBottom();
+    if (atBottom) {
+      scrollToBottom();
+    }
+
```
Comment (blamarche) [inline static/index.js:465]: I think for this to be done 'clean', scrollToBottom() should be added to appendChatMessage and appendSystemMessage, and that logic could be added in there as well. Also, have you tested this still works when using a mobile device and the keyboard is pulled up, etc?

**Comment by pageben** [comment]
scrolltobottom and clearbtn code has been updated based on suggestions from blamarche.  This code has not been tested on mobile devices.

**Comment by pageben** [comment]
Update: I checked it on android with the keyboard visible and without.  Works exactly as expected both ways.  If not scrolled to bottom, new messages won't yank the view back to the bottom.  If at the bottom, new messages append and the view scrolls so they're visible.

**Comment by pageben** [comment]
On Android, I'm unable to resize textarea#m.  The resize handle at the bottom right is either too small to manipulate or it's disabled.  It's probably not an issue.  I'm not sure mobile devices need this functionality.

**Pull request merged**